app/models/miq_storage_metric.rb

```diff
@@ -96,7 +96,7 @@ def metrics_rollups_by_statistic_time(rollup_type, statistic_time)
   end
 
   def metrics_rollups_by_rollup_type(rollup_type)
-    miq_metrics_rollups.where(rollup_type => rollup_type).to_a
```

Comment:
@kbrock  It looks like this class is abstract and the subclasses such as `OnTapDiskMetric` implement the `has_many miq_metrics_rollups` so it's a matter of creating 1 OnTapDiskMetric with 2 OnTapDiskMetric::METRICS_ROLLUP_CLASS_NAME instances and call this method on the OnTapDiskMetric instance.